documentation/SpecialRules.md

```diff
@@ -0,0 +1,8 @@
+### Special Rules (SA0000-)
+Rules which provide special functionality like workarounds, configuration errors, etc.
+
+Identifier | Name | Description
+-----------|------|-------------
+[SA0000](SA0000Roslyn7446Workaround.md) | Roslyn7446Workaround | Workaround incomplete diagnostics in Visual Studio 2015 Update 1.
+[SA0001](SA0001.md) | XmlCommentAnalysisDisabled | All diagnostics of XML documentation comments has been disabled due to the current project configuration.
+[SA0002](SA0002.md) | InvalidSettingsFile | The <em>stylecop.json</em> settings file could not be loaded due to a deserialization error.
```

Comment:
:question: Can we not use `**stylecop.json**` instead of `<em>stylecop.json</em>`?